src/antonia2203/button.pug

```diff
@@ -0,0 +1,10 @@
+html 
+  head 
+    title Button Tag 
+  body(
+    style="text-align:center; background-color: pink"
+  )
+    h1 The button Element 
```

Comment:
nesting a heading element inside of a button element will cause failures.

please check this link as well:
https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-heading-inside-button.md
<img width="1152" alt="image" src="https://user-images.githubusercontent.com/78360817/188498437-7014f740-e6af-48be-8f77-d5a6dcc03564.png">